cocos2d/render-texture/CCRenderTextureCanvasRenderCmd.js

```diff
@@ -88,6 +88,7 @@
 
         var scale = cc.contentScaleFactor();
         cc.renderer._renderingToCacheCanvas(this._cacheContext, node.__instanceId, scale, scale);
+        node.sprite._renderCmd.setDirtyFlag(cc.Node._dirtyFlags.colorDirty);
```

Comment:
Given the current mechanism of dirty system. textureDirty will not trigger the update, then the dirty flag is not set correctly.